lib/dataSources.js

```diff
@@ -38,15 +38,18 @@ const add = (conn, name, options) => {
   }).then(httpBody);
 };
 
-const update = (conn, name, options) => {
+const update = (conn, name, options, requestOptions = {}) => {
   const headers = conn.headers();
   headers.set('Content-Type', 'application/json');
+
+  const body = { name, options };
+  if (requestOptions.force) {
+    body.force = true;
```

Comment:
i would set this to be the value of the option rather than always true when present (so this way if you have it as `requestOptions.force = false` it passes it through rather than making it true)